### PR TITLE
add raven english docs

### PR DESCRIPTION
**Pull request by luckymrwang**

### CODE REVIEW

File: docs/core-concepts/raven.md [hunk 1]
```diff
@@ -2,67 +2,66 @@
...
 
-- **Raven Controller Manager**：标准的Kubernetes控制器，以Deployment的方式部署在部分云上节点中，负责监控边缘节点状态，为每一个边缘节点池选取一个跨边流量的出口作为gateway node，并能够在当前gateway node失活的情况下完成gateway node的切换，所有的跨边流量都将由各个边缘节点池的gateway node完成转发；
-- **Raven Agent**：以DaemonSet的方式部署，运行在集群的每一个节点，它根据每个节点的角色（gateway or non-gateway）在节点上配置路由信息或VPN隧道信息；
+- **Raven Controller Manager**：
+The native Kubernetes controller is deployed in some nodes on the cloud as a `Deployment`, monitoring the status of edge nodes, selecting an egress for cross-edge traffic as a gateway node for each edge node pool. When the current gateway node is dead and other node will be switched. All cross-edge traffic will be forwarded by the gateway node of each edge node pool;
+
+- **Raven Agent**：
+It is deployed as a `DaemonSet` and runs on each node of the K8s cluster. It configures route or VPN tunnel on the node according to the role of each node (gateway or non-gateway);
 
 上述两个组件通过一个[Gateway CRD](https://github.com/openyurtio/raven-controller-manager/blob/main/pkg/ravencontroller/apis/raven/v1alpha1/gateway_types.go) 来交换配置路由和建立VPN隧道的必要信息，如下图所示：
```
Comment (rambohe-ch) [inline docs/core-concepts/raven.md:23]: please translate the content to english.

File: docs/core-concepts/raven.md [hunk 2]
```diff
@@ -2,67 +2,66 @@
...
+## 3. Features and Advantages
 
-## 3. 特性及优势
-
-特性：
+Features:
 
-- 无侵入：对原生的K8s CNI网络无侵入，仅劫持跨边流量进行转发
-- 安全：使用成熟稳定的IPsec技术对跨边流量进行加密
+- No intrusion: 对原生的K8s CNI网络无侵入，仅劫持跨边流量进行转发No intrusion: No intrusion into the native K8s CNI network, only cross-edge traffic is hijacked for forwarding
```
Comment (rambohe-ch) [inline docs/core-concepts/raven.md:36]: please delete `No intrusion: 对原生的K8s CNI网络无侵入，仅劫持跨边流量进行转发`

**Comment by rambohe-ch** [comment]
@luckymrwang please update doc: versioned_docs/version-v0.7.0/core-concepts/raven.md

**Comment by luckymrwang** [comment]
@rambohe-ch The latest code has been commited.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**